Safety/Safety-Systems/GIS.rst

```diff
@@ -1,6 +1,325 @@
+
+.. _Safety-Systems-GIS-Main-Controller-Interface:
+
+Troubleshooting
+================
+
+In order to troubleshoot you need to understand the structure and check the alarm logs, at the bottom of the Main panel interface.
+
+.. admonition:: Important
+
+    Beware that sometimes the detection disappears, this means the interlocks are no longer triggered (because someone may have reset it) but the action is still in place.
+
+
+.. _Safety-Systems-GIS-Dealing-with-interlocks-in-GIS-Main-Control-Panel:
+
+Dealing with interlocks in GIS Main Control Panel
+--------------------------------------------------
+
+Remember, depending on the :ref:`connection type <Safety-Systems-GIS-connection-type>`
+
+* **Wired subsystems** are :guilabel:`RESET` at the Main Control Panel.
+
+* **Network-based systems** require resetting within their corresponding subsystem to clear the interlock in the Main Control panel.
+
+
+
+.. _Safety-Systems-GIS-Top-end-system:
+
+**Top End systems** (M2CAM), a special case
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The status listed does NOT indicate the status of each individual element, but rather the **status of the E-Stop circuit** (including reset action) for each element.
+
+After an **STO or power cycle** of the Top End systems, you must :guilabel:`RESET` the GIS E-stop status buttons before enabling the system from the corresponding interface. Even if the lights are green (indicating that the E-stop circuit is okay), the :guilabel:`RESET` **action is still required**.
+
+.. This maybe change to an autoreset action.
+
+
+
+If another interlock triggered you need to follow the corresponding procedure to fix it, shown in table below. 
+
+.. we may want to add links to the procedures.
+
+.. list-table:: How to fix a triggered interlock
+ :widths: 20 25 
+ :header-rows: 1
+
+ * - GIS signal
+   - Do
+ * - D2 Earthquake alarm
+   - *Auto reset (when earthquake stop)*
+ * - D3 GIS Internal failure
+   - Power cycle GIS cabinet
+ * - D4 GIS Estops
+   - Reset triggered Estop at GIS panel
+ * - D5 Unauthorized Pier access
+   - Close 6th level safety gate
+ * - D5 Unauthorized Dome access
+   - Close 7th level safety gate (bypassed, July 2024)
+ * - D7, D8, D9, D17 (TMA signals)
+   - Reset at TMA EUI
+ * - D10 to D13 (Dome signals)
+   - Reset at Dome EUI
+ * - D14 Camara Rotator Pin inserted
+   - Reset at Camara Rotator EUI
+ * - D15, D16 (Pflow signals)
+   - *Auto reset (when lift position changes)*
+ * - D18 M1M3 Interlock
+   - Reset at M1M3IS
+ * - D19 Man-lift parked
+   - *Not currently used (July, 2024)*
+ * - D20 M2 mirror in *closed-loop*
+   - *Auto reset when M2 is in *closed-loop* again
+
+
+.. admonition:: Safety gates operation
+
+    * To open a gate: Press :guilabel:`release` button (gate is open), then move the handle.
+    * To close a gate: move the handle to close the gate. Press the :guilabel:`lock` button (the gate is closed).
```

Comment:
You may capitalize Move here as it's a complete sentence.

Reply:
Fixed, I also capitalized the buttons Release and Lock as in the AuxTel gate are written as such.